DESCRIPTION

```diff
@@ -16,7 +16,7 @@ Imports:
     shiny (>= 1.0.0)
 Remotes:
     JGCRI/rgcam,
-    JGCRI/gcammaptools
+    JGCRI/gcammaptools@alter-proj4-method
```

Comment:
We will need to change this once we've merged the alter-proj4-method branch on gcammaptools.

I recommend we change this back to just JGCRI/gcammaptools, and deal with JGCRI/gcammaptools#27 before we merge this one.